examples/advanced/nlp-ner/jobs/bert_ncbi/app/custom/utils/data_sequence.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+import torch
+
+
+def align_label(
+    tokenized_inputs,
+    origional_text,
+    labels,
+    labels_to_ids,
+    tokenizer=None,
+):
+    null_label_id = -100
```

Comment:
Should this be called `pad_token` to be consistent with the Learner?

Reply:
yes, changed, and set as a parameter to be fed by base learner